### DISCUSSION TITLE
Download command

### DISCUSSION THREAD

**Discussion by Livijn** [spatie/laravel-backup#1284] [Ideas]
I was thinking about creating a command for downloading the latest SQL backup from production. This way you can debug and test stuff with production data easily.

`php artisan backup:download-latest`
or something like
`php artisan backup:fetch`

Note that it would be important that the file was added to .gitignore.

**Comment by freekmurze** [comment]
Thanks for the suggestion, but I think this is out of scope for the package.

**Reply by Livijn** [reply]
Okay, maybe i'll create a separate package for it instead.
